docs/docs/development/build-flash.md

```diff
@@ -90,6 +90,10 @@ This produces `left` and `right` subfolders under the `build` directory and two
 
 ### Building from `zmk-config` Folder
 
+Getting your `zmk-config` to be used by your build environment differs based on what type of build environment you are using.
+
+#### non-containerized environment
```

Comment:
This needs proper capitalization, also for anyone that isn't a developer, or doesn't know what a "containerized environment" is, this might be really confusing.

Perhaps something like:

```suggestion
#### Local/Host Build Environment.
```